contracts/contracts/RequestManager.sol

```diff
@@ -211,7 +211,7 @@ contract RequestManager is Ownable {
         );
         require(request.activeClaims == 0, "Active claims running");
 
-        request.depositReceiver = request.sender;
+        request.withdrawInfo = BeamerUtils.FillInfo(request.sender, bytes32(0));
```

Comment:
Wouldn't it be enough to just set the filler here?

Reply:
totally right